roles/zuul/files/jobs/bonnyci.yaml

```diff
@@ -10,7 +10,10 @@
               copy-after-failure: true
 
 - job-template:
-    name: 'bonnyci-run-{pipeline}'
+    node: 'ubuntu-xenial'
```

Comment:
Does this work as a default value? I think that this needs to be `node: {node}` in order to use the parameter passed below.